packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -20,6 +20,12 @@ const DEFAULT_CONFIG: Config = {
     }
 };
 
+const CLOSED_ERROR_TEXTS: Array<string> = [
+    'Promise was rejected with the following reason: no such window: target window already closed\n' +
```

Comment:
text looks too specific and may stop works after chrome/selenium version change.